airsonic-main/src/main/webapp/WEB-INF/jsp/podcastChannel.jsp

```diff
@@ -166,21 +292,51 @@
 
 </table>
 
-<table style="padding-top:1em"><tr>
-    <c:if test="${model.user.podcastRole}">
-        <td style="padding-right:2em"><div class="forward"><a href="javascript:downloadSelected()"><fmt:message key="podcastreceiver.downloadselected"/></a></div></td>
-        <td style="padding-right:2em"><div class="forward"><a href="javascript:deleteSelected()"><fmt:message key="podcastreceiver.deleteselected"/></a></div></td>
-    </c:if>
-    <td style="padding-right:2em"><div class="forward"><a href="javascript:refreshPage()"><fmt:message key="podcastreceiver.refresh"/></a></div></td>
-    <c:if test="${model.user.adminRole}">
-        <td style="padding-right:2em"><div class="forward"><a href="podcastSettings.view?"><fmt:message key="podcastreceiver.settings"/></a></div></td>
+<select id="moreActions" class="pagetype-dependent type-album type-video" onchange="actionSelected(this.options[selectedIndex].id);" style="margin-bottom:1.0em">
+    <option id="top" selected="selected"><fmt:message key="playlist.more"/></option>
+    <option id="selectAll">&nbsp;&nbsp;<fmt:message key="playlist.more.selectall"/></option>
+    <option id="selectNone">&nbsp;&nbsp;<fmt:message key="playlist.more.selectnone"/></option>
+    <c:if test="${model.user.downloadRole}">
+        <option id="download">&nbsp;&nbsp;<fmt:message key="common.download"/></option>
```

Comment:
This "Download" and the "Download" outside the select box is now confusing.

They both mean different types of downloads

Reply:
This existing "Download" button was very confusing for me since started using Airsonic. I thought that the "Download" would start a download to the client, but it actually starts Airsonic to download the podcast to the server. Related issue: https://github.com/airsonic/airsonic/issues/1479

What do you think about renaming the "Download" outside the select box to "Download to server"?

Reply:
sure, sounds good to me